tailwind.config.js

```diff
@@ -19,6 +19,7 @@ try {
 let themeColors = {
   primary: uiColors.indigo,
   secondary: uiColors.purple,
+  'cat-foobar': '#0800ff'
```

Comment:
This has to be added, as it seems it has to be declared in order to be 'overridden' by the same key:pair value in `theme/tailwind.config.yml`